bassa

```diff
@@ -0,0 +1,125 @@
+#!/bin/bash
+: '
+# python3 Main.py
+# gulp serve
+# arie2c --enable-rpc
+'
+start(){
+	# first check the current status of the server using log file exsitance
+	check_status=$( status )
+	if [ "$check_status" == "inactive" ]
+	then
+		echo -ne '\e[32mstarting...'
+		# this one is just to look good 
+		for number in {1..6}
+		do
+			echo -ne '.'
+			sleep .5
+		done
+		# get the base directory of the bassa server files
+
+		BASEDIR=$(dirname "$0")
+		PYSERVER="$BASEDIR/components/core/Main.py"
+		UISERVER="$BASEDIR/ui"
+
+		# this one is to use the temporary folder for extra data
+
+		if [ ! -d "/tmp/bassa" ]; then
+			mkdir /tmp/bassa
+		fi
+
+		# making log file for server
+
+		if [ ! -f "$BASEDIR""/python-server.log" ]; then
+			echo "" > "$BASEDIR""/python-server.log"
+			echo "" > "$UISERVER""/gulp-serve.log"
+			echo "" > "$UISERVER""/aria2c.log"
+		fi
+		
+		# making process in backgroud using nohup
+
+		nohup python3 "$PYSERVER" &>> "$BASEDIR""/python-server.log" &
+		cd "$UISERVER"
+		nohup gulp serve &>> "$UISERVER""/gulp-serve.log" &
+		nohup aria2c --enable-rpc &>> "$UISERVER""/aria2c.log" &
+		echo -e 'started successfully\e[0m'
+	else
+		echo -e "\e[32mServer is running successfully\e[0m"
+	fi
+}
+stop(){
+	echo -ne '\e[31mstoping.'
```

Comment:
a minor typo. Should be stopping or terminating

Reply:
What typo you are thinking?